### DISCUSSION TITLE
release version download

### DISCUSSION THREAD

**Discussion by thploeg** [robinraju/release-downloader#695] [General]
Hi! I am kinda new in github actions. I used the release-downloader and get the latest version. The problem and error appearing is: Eroor: there are no assets found.
The problem is, there are assets. Is there a solution for this?

**Comment by thploeg** [comment]
with version 1.8, it works.

**Reply by Vaishno23** [reply]
I am also facing the same issue, does any  one got the fix ?

**Reply by thploeg** [reply]
it still doesnt work, but you can use version 1.8 or use the checkout downloader, when a release is made. If you need help, let me know.

**Reply by Vaishno23** [reply]
Thanks much for your respond, 

our requirement is used v1.9 only , it would be grete help if you can guide how to use  checkout downloader!

**Reply by thploeg** [reply]
It depends, where do you want to use it for? 
I use a trigger that triggers on release for my workflow. After that I use actions/checkout@V3 to checkout my repo under ${GITHUB_WORKSPACE}. You can acces your latest download. If you want to download only the latest release on your own trigger, you have to use version 1.8. 
Let me know if you have difficulties.

**Reply by Vaishno23** [reply]
Many thanks:)

As you mentioned, "If you want to download only the latest release on your own trigger, you have to use version 1.8." Is there any documentation you could refer me to where I could find out this information, or can I investigate whether there is a problem with version 1.9? I wanted to download only the most recent release, and version 1.8 worked well for me.

**Reply by thploeg** [reply]
I've found out myself that version 1.8 is compatible to get the assets you need, the 1.9 version is a problem, I know where the problem is and wanted to solve it with pull request, but I've time too short. 
If I make the documentation and solved the problem, I will let you know.

**Reply by Vaishno23** [reply]
Thanks much

**Reply by thploeg** [reply]
Can I ask you, what trigger do you want to use?
I recommend, use version tags to trace those releases.

**Reply by Vaishno23** [reply]
This what i am using .

name: Dummy Release
uses: robinraju/release-downloader@v1.9
with:
tag: ${{ xyx 'latest' }} # If tag is not provided, default to 'latest'
fileName: "*.zip"
token: ${{ xyx }}
latest: true # Always set to true to download the latest release if tag is not provided or is invalid
tarBall: true
zipBall: true

My aim is to remove the below warning. I was using my previous workflow version v.7. Then, I used version v.8 With both versions, I was receiving the below warning, which I removed(used with v.9 version) . However, when I used version v.9, I was receiving the there are no assets found error.

warning as below
repository: xyxNode.js 16 actions are deprecated. Please update the following actions to use Node.js 20: robinraju/release-downloader@v1.8. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
